examples/fineFoods/client/src/components/numberInput.tsx

```diff
@@ -10,9 +10,22 @@ export const NumberInput: React.FC<NumberInputProps> = ({
     setValue,
 }) => {
     return (
-        <div className="relative overflow-hidden">
+        <div className="relative overflow-hidden flex items-center">
+            <button
+                className="h-full border p-2 transition-all hover:bg-gray-50 active:bg-gray-50 rounded-tl-md rounded-bl-md"
+                onClick={() =>
+                    setValue((prev) => {
+                        if (prev > 0) {
+                            return prev - 1;
+                        }
+                        return 0;
+                    })
+                }
+            >
+                <ChevronDownIcon className="text-primary h-full w-4" />
```

Comment:
```suggestion
                <ChevronDownIcon className="text-primary h-4 w-4" />
```

Can we also change this? It's breaking on safari 😅